src/Hocon.Configuration.Test/Hocon.Configuration.Tests.csproj

```diff
@@ -19,8 +22,12 @@
     <ProjectReference Include="..\Hocon.Configuration\Hocon.Configuration.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
     <Reference Include="System.Configuration" />
   </ItemGroup>
 
+  <Target Name="CopyCustomContent" AfterTargets="AfterBuild">
+    <Copy SourceFiles="app.config" DestinationFiles="$(OutDir)\testhost.dll.config" />
```

Comment:
@Arkatufus change this to `App.config` with a capital A and that should fix the build on Linux